SwiftUI_DZ/SwiftUI_DZ/AutorizeView/AutorizeView.swift

```diff
@@ -140,28 +157,37 @@ struct AutorizeView: View {
                 .frame(width: Constants.textFieldWidthDouble, height: 70, alignment: .leading)
                 .font(.system(size: Constants.fontSizesDouble.1))
                 .padding(.all)
-            SecureField(Constants.passwordPlaceholderString, text: $autorizeViewModel.passwordText)
-                .focused($isPasswordFocused)
-                .onChange(of: autorizeViewModel.passwordText) { text in
-                    autorizeViewModel.charactersCount = text.count
-                    autorizeViewModel.setPasswordText(text)
-                    autorizeViewModel.setInSignFlag()
+            ZStack {
+                SecureField(Constants.passwordPlaceholderString, text: $autorizeViewModel.passwordText)
+                    .focused($isPasswordFocused)
+                    .onChange(of: autorizeViewModel.passwordText) { text in
+                        autorizeViewModel.charactersCount = text.count
+                        autorizeViewModel.setPasswordText(text)
+                        autorizeViewModel.setInSignFlag()
+                    }
+                    .font(.system(size: Constants.fontSizesDouble.1))
+                    .frame(width: Constants.textFieldWidthDouble)
+                    .padding(.all)
+                    .keyboardType(.numberPad)
+                    .focused($isPasswordFocused)
+                    .modifier(ShakeEffect(animatableData: CGFloat(autorizeViewModel.sizeAnimation)))
```

Comment:
```suggestion
                SecureField(Constants.passwordPlaceholderString, text: $autorizeViewModel.passwordText)
                    .focused($isPasswordFocused)
                    .onChange(of: autorizeViewModel.passwordText) { text in
                        autorizeViewModel.charactersCount = text.count
                        autorizeViewModel.setPasswordText(text)
                        autorizeViewModel.setInSignFlag()
                    }
                    .font(.system(size: Constants.fontSizesDouble.1))
                    .frame(width: Constants.textFieldWidthDouble)
                    .padding(.all)
                    .keyboardType(.numberPad)
                    .focused($isPasswordFocused)
                    .modifier(ShakeEffect(animatableData: CGFloat(autorizeViewModel.sizeAnimation)))
```
я б конечно всё повыносила в отдельные свойства